Mail/Views/Thread List/ThreadListView.swift

```diff
@@ -44,7 +44,8 @@ struct ThreadListView: View {
     @AppStorage(UserDefaults.shared.key(.threadDensity)) private var threadDensity = DefaultPreferences.threadDensity
     @AppStorage(UserDefaults.shared.key(.accentColor)) private var accentColor = DefaultPreferences.accentColor
 
-    @State private var isShowingComposeNewMessageView = false
+    /// Assign a draft will open an edit modal for it.
+    @State private var newDraft: Draft?
```

Comment:
If variable needs a comment to describe it's function maybe rename it something more meaningful ? eg.
```suggestion
    @State private var newPresentedDraft: Draft?
```
Because views in SwiftUI are function of the state we directly understand that if `newPresentedDraft` is not nil something will be currently presented

Reply:
Sure, I like `newPresentedDraft`